vmm.mk

```diff
@@ -30,7 +33,14 @@ ifeq ($(strip $(SDDF)),)
     $(error libvmm needs the location of the SDDF to build virtIO components)
 endif
 
+# we need ${SDDF} for virtIO; we need ${LIBVMM} for all
+# the libvmm api interfaces
+ifeq ($(findstring ${SDDF}/include, ${CFLAGS}),)
 CFLAGS += -I${SDDF}/include
+endif
+ifeq ($(findstring ${LIBVMM_DIR}/include,${CFLAGS}),)
+CFLAGS +=-I${LIBVMM_DIR}/include
```

Comment:
Is this missing a space between `+=` and `-I`?

Reply:
Cosmetic space, yes